judge/contest_format/ioi.py

```diff
@@ -49,10 +49,10 @@ def update_participation(self, participation):
                                              .filter(points=Subquery(
                                                  participation.submissions.filter(problem_id=OuterRef('problem_id'))
                                                                           .order_by('-points').values('points')[:1]))
-                                             .annotate(time=Min('submission__date'))
-                                             .values_list('problem_id', 'time', 'points', 'problem__points'))
+                                             .annotate(time=Min('submission__date'), disqualified=Max('is_disqualified'))
```

Comment:
If no submissions have `is_disqualified = True`, then it will be false. Basically, we mark the participation's problem as "disqualified" if any of the user's max submissions have `is_disqualified = True`

Reply:
Oh, this looks at multiple submissions, nvm.